offerzen/global/company-fast-track-access/company-prospect-form@v1.2.js

```diff
@@ -0,0 +1,241 @@
+(function () {
+  // FORM TARGETTING
+  const prospectFormId = '#Company-Prospect-Form';
+  const pageRef = 'onSubmitCompanyProspectForm';
+  const formSubmitButtonSelector = '#submit-button';
+  const formContainerClass = '.js-company-prospect-form';
+  // END FORM TARGETING
+  // This needs to run regardless of whether JQuery has loaded
+  // Disable button
+  const formSubmitButton = document.querySelector(
+    `${prospectFormId} ${formSubmitButtonSelector}`
+  );
+
+  // Store button labels for state changes
+  let initialButtonValue = formSubmitButton.getAttribute('data-initial');
```

Comment:
```suggestion
  let initialButtonValue = formSubmitButton.value();
```
Should also work, reduces the need for extra attributes